src/components/FeedbackCard.js

```diff
@@ -39,23 +40,34 @@ const ButtonContainer = styled.div`
 
 const FeedbackCard = () => {
   const [feedbackSubmitted, setFeedbackSubmitted] = useState(false)
+  const [isHelpful, setIsHelpful] = useState(false)
 
-  const title = feedbackSubmitted
-    ? "Thanks for your feedback!"
-    : "Did this page help answer your question?"
+  const title = isHelpful ? (
+    <>Thanks for your feedback!</>
+  ) : (
+    <>
+      Join our public <Link to="https://discord.gg/rZz26QWfCg">Discord</Link>{" "}
+      and get the answer you're looking for
```

Comment:
Suggested some punctuation for the negative feedback response - other than that LGTM 👍

```suggestion
      Join our public <Link to="https://discord.gg/rZz26QWfCg">Discord</Link>{" "}
      and get the answer you're looking for.
```